### ISSUE TITLE
I found that there is no moving_average_mean/var, which is necessary in BatchNorm. Is it ok?

### ISSUE THREAD

**Issue by clks-wzz** [shaohua0116/Group-Normalization-Tensorflow#2]
Hello, sir. I found that there is no moving_average_mean/var, which is necessary in BatchNorm. Is it ok for GroupNorm?

**Comment by Bingohong** [comment]
i found the same problem there is no moving_average process. how is to compute gn in test step?

**Comment by Bingohong** [comment]
@shenekeng853 i add the ema process, maybe you can try it https://github.com/Bingohong/GroupNormalization-tensorflow/blob/master/README.md
